components/organisms/Contributors/contributors.tsx

```diff
@@ -35,8 +38,22 @@ const Contributors = ({ repositories }: ContributorProps): JSX.Element => {
   const router = useRouter();
   const limit = router.query.limit as string;
   const topic = router.query.pageId as string;
+  const { range = 30 } = router.query;
 
   const { data, meta, setPage, isError, isLoading } = useContributors(Number(limit ?? 10), repositories);
+
+  // New contributors
+  const { meta: currentNewContributorsMeta } = useInsightsContributors(10, repositories, "new");
+  const { meta: prevNewContributorMeta } = useInsightsContributors(10, repositories, "new", 30);
```

Comment:
Hardcoded the limit as 10 because we are not making use of the data in reality; all we need is the meta to get the count.